src/room_member.cpp

```diff
@@ -256,12 +256,24 @@ bool RoomMember::IsConnected() const {
     return state == State::Joining || state == State::Joined;
 }
 
+void RoomMember::CloseConnection() {
+    peer->Shutdown(300);
+    member_information.clear();
+    nickname.clear();
```

Comment:
I personally like the std::string::clear (looks like resetting something) better, any reason why you want `= ""` (looks like a name change) ?